example/example/tests/test_grapple.py

```diff
@@ -59,9 +60,11 @@ def test_pages(self):
 
         executed = self.client.execute(query)
 
-        self.assertEquals(type(executed["data"]), dict_type)
+        self.assertTrue(isinstance(executed["data"], collections.OrderedDict))
```

Comment:
This started failing after using `requirements-channels.txt` file for dependencies with 
```
    self.assertEquals(type(executed["data"]), dict_type)
AssertionError: <class 'collections.OrderedDict'> != <class 'dict'>
```

Let me know if this need to be addressed in another PR

Reply:
That has to do with graphql-core versions, I believe. 

https://github.com/GrappleGQL/wagtail-grapple/blob/cdc0d1d94a5ac7107e8563929ea185db5bce3787/example/example/tests/test_grapple.py#L9-L12, via https://github.com/GrappleGQL/wagtail-grapple/pull/105/commits/2313c52c41796075c26ac6f1159b4f1f75fd2ef7

Reply:
It does, I reverted this change